### ISSUE TITLE
Extra ext appended to files copied to file:///

### ISSUE THREAD

**Issue by ranlu** [seung-lab/cloud-files#92]
```
cloudfiles cp test.txt /tmp
```
create /tmp/test.txt
```
cloudfiles cp test.txt file:///tmp
```
create /tmp/test.txt.txt, any ext will be double in my test.

**Comment by ranlu** [comment]
I must got confused yesterday, both case gives a `test.txt.txt` with cloud-files 4.17.2 installed from pip

**Comment by william-silversmith** [comment]
Ah, I think this bug was introduced to try to compensate for compression extensions. Will fix this...

**Comment by ranlu** [comment]
Seems to be fixed since 4.18.0
